src/scripts/clipperUI/panels/successPanel.tsx

```diff
@@ -25,24 +40,47 @@ class SuccessPanelClass extends ComponentBase<{ }, ClipperStateProp> {
 		}
 	}
 
+	private showRatingsPrompt(): any[] {
+		if (this.props.clipperState.shouldShowRatingsPrompt) {
+			let message: string = RatingsHelper.getMessage(this.state.userSelectedRatingsPromptStage);
+			let buttons: DialogButton[] = RatingsHelper.getDialogButtons(this);
+
+			if (!Utils.isNullOrUndefined(message)) {
+				let animationStrategy = RatingsHelper.getAnimationStategy(this);
+
+				return (
+					<DialogPanel
```

Comment:
We could create a RatingsPanelClass that renders a DialogPanel to make things a bit cleaner.